packages/mdc-select/mdc-select.scss

```diff
@@ -188,8 +188,7 @@ $mdc-select-menu-transition: transform 180ms $mdc-animation-standard-curve-timin
   }
 }
 
-.mdc-select--disabled,
```

Comment:
Just leaving a note here that the `@each` after this will be removed in #2098 anyway, so it probably doesn't merit touching in this PR.